lua/browse/bookmarks.lua

```diff
@@ -38,12 +42,25 @@ M.search_bookmarks = function(config)
             ordinal = entry
         elseif type(entry) == "table" and type(entry[2]) ~= "table" then
             value = entry[2]
-            display = entry[1] .. " -> " .. value
-            ordinal = value
+            display = entry[1] .. "  " .. value
```

Comment:
I am not sure about the accessibility of the icon. Since not all of the users will be having nerd fonts installed. Can we use some other icon or something else which will be available by default to everyone else.